styles.css

```diff
@@ -0,0 +1,696 @@
+    gap: 16px;
+    position: relative;
+    top: 16px;
+  }
+
+  .navigation-item {
+    list-style: none;
+    display: flex;
+    justify-items: center;
+    width: 67px;
+  }
+
+  .navigation-item-text {
+    width: 63px;
+    height: 20px;
+    font-family: "inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    text-decoration: none;
+    color: #344563;
+  }
+
+  #toolbar-menu-button {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    position: absolute;
+    width: 40px;
+    height: 40px;
+    right: 24px;
+    top: 2px;
+    border-radius: 8px;
+  }
+
+  #toolbar-menu-icon {
+    width: 24px;
+    height: 24px;
+    position: relative;
+    justify-content: center;
+    align-items: center;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    top: 20px;
+    left: 8px;
+    border: none;
+  }
+
+  #toolbar-menu-union-icon,
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    background-color: #000;
+    position: absolute;
+    width: 100%;
+    height: 2px;
+  }
+
+  #toolbar-menu-union-icon::before,
+  #toolbar-menu-union-icon::after {
+    content: "";
+  }
+
+  #toolbar-menu-union-icon::before {
+    transform: translate(0, 10px);
+  }
+
+  #toolbar-menu-union-icon::after {
+    transform: translate(0, -10px);
+  }
+
+  /* Headline section */
+  #headline {
+    height: auto;
+    padding: 160px 0;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .headline-heading {
+    width: 100%;
+    font-size: 300%;
+  }
+
+  #color-this {
+    color: green;
+    letter-spacing: 3px;
+  }
+
+  #color-this::before {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .headline-description {
+    justify-self: center;
+    letter-spacing: 0.05em;
+  }
+
+  #headline-social-media {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .social-media {
+    display: flex;
+    gap: 24px;
+  }
+
+  .bottom-icon-container {
+    width: 40px;
+    height: 40px;
+    border: 1px solid #dfe1e6;
+    border-radius: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .headline-to-bottom-icon {
+    height: 114px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .bottom-icon-container:hover {
+    background-color: #36b37f;
+  }
+
+  /* WORKS SECTION */
+  #works {
+    width: 81%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 80px 0;
+    gap: 48px;
+  }
+
+  #works-title {
+    width: 100%;
+    height: 48px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .works-title-text {
+    width: 100%;
+  }
+
+  #work-cards {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 98%;
+  }
```

Comment:
- [ ] Kindly consider using `display: grid` instead of flex here to layout your cards. in the main container of cards using grid is the project requirement. following is the guideline to how to layout your projects cards.
  -  In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.